frame/executive/src/lib.rs

```diff
@@ -144,10 +150,14 @@ impl<
 		WeighBlock<System::BlockNumber>,
 > Executive<System, Block, Context, UnsignedValidator, AllModules>
 where
-	Block::Extrinsic: Checkable<Context> + Codec,
+	Block::Extrinsic: Checkable<Context> + UnsafeConvert<Context> + Codec,
```

Comment:
```suggestion
	Block::Extrinsic: Checkable<Context> + UnsafeConvert<Context, UnsafeResult = CheckedOf<Block::Extrinsic, Context>> + Codec,
```

Than you should be able to remove the other bounds below.